charts/pravega-operator/values.yaml

```diff
@@ -22,3 +22,6 @@ crd:
 
 # whether to enable test mode
 testmode: false
+
+# whether to run in production environment
+production: true
```

Comment:
What is this for?

Reply:
This value is to determine which of the 2 config-maps to deploy : the one for production env or the one for test env.

Reply:
That can be done using `testmode` variable.